### DISCUSSION TITLE
How to update a container when updating a configuration/.env file?

### DISCUSSION THREAD

**Discussion by Shaked** [containrrr/watchtower#1379] [General] [answered]
I have a docker image that uses environment variables. 

When I start the container, I use an .env variable to set them up. 

Sometimes I change the configuration, without updating the image of course, and I'd like the container to restart, ideally with no downtime, with the newly update configuration. 

How would you do that? 

Thank you

**Comment by simonaronsson** [comment, marked as answer]
I'm sorry, that is unfortunately not something we support at this point.
